lib/sycamore/sycamore/tests/integration/connectors/opensearch/test_pdf_to_opensearch.py

```diff
@@ -92,33 +97,67 @@ def test_pdf_to_opensearch():
 
         """
 
-    paths = str(TEST_DIR / "resources/data/pdfs/")
+    s3_cache_base_path = os.environ["SYCAMORE_S3_TEMP_PATH"]
```

Comment:
Is this a net new variable we need for the integ tests? Is the plan to add it as a variable/secret in github.

Reply:
I stole it from `test_file_writer_to_s3.py` https://github.com/aryn-ai/sycamore/blob/472d22e116f1ef2e1250ae7221a19fb2dba27ab7/lib/sycamore/sycamore/tests/integration/connectors/file/test_file_writer_to_s3.py#L67